src/components/AdminRegisterPage/index.jsx

```diff
@@ -28,7 +27,7 @@ const AdminRegisterPage = ({ match }) => {
         // user is authenticated but doesn't have the `enterprise_admin` JWT role; force a log out so their
         // JWT roles gets refreshed. on their next login, the JWT roles will be updated.
         const logoutRedirectUrl = getLogoutRedirectUrl(getProxyLoginUrl(enterpriseSlug));
-        history.push(logoutRedirectUrl);
+        getAuthenticatedHttpClient().get(logoutRedirectUrl);
```

Comment:
Are we now using `getAuthenticatedHttpClient.get()` (aka axios instance) to trigger a redirect? Feels a little strange since we're not really making an API request here.

Should we revert back to using `global.location.href = logoutRedirectUrl`?